src/controllers/checkins.ts

```diff
@@ -7,20 +7,34 @@ import { parse_mag_stripe } from '../utils/card';
 
 const EVENT_SHEET_ID = "EVENTS";
 const CHECKIN_SHEET_ID = "CHECKINS";
+const RPI_SHEET_ID = "RPI";
 
 export const create_check_in: Express.RequestHandler = async (req, res) => {
   let check_in: CheckInInput = req.body;
   if (check_in.student_id.charAt(0) === ";") check_in.student_id = parse_mag_stripe(check_in.student_id);
 
   const sheet = google.sheets("v4");
 
+  const read_result = await sheet.spreadsheets.values.get({
+    spreadsheetId: process.env.SHEET_ID,
+    auth: sheet_auth(),
+    range: `${RPI_SHEET_ID}!A1:C`
+  });
+
+  const rows = read_result.data.values as string[][];
+  const ser: Array<RPI> = serialize_rows(rows) as Array<RPI>;
```

Comment:
Reminder that the alias will be populated by the HEARTBEAT not by this ping. Nothing you have to change, documentation purposes.